File: MonoDevelop.UnitTesting.Expecto/ExpectoTestSuite.fs
namespace MonoDevelop.UnitTesting.Expecto

open System
open MonoDevelop.Projects
open MonoDevelop.UnitTesting

type ExpectoTestSuite =
    inherit UnitTestGroup

    new(name) = { inherit UnitTestGroup(name) }
    new(name, ownerSolutionItem) = { inherit UnitTestGroup(name, ownerSolutionItem) }

    override this.OnRun testContext =
        
        raise (new NotImplementedException())

File: MonoDevelop.UnitTesting.Expecto/ExpectoTestSuite.cs
using System;
using MonoDevelop.Projects;
using MonoDevelop.UnitTesting;

namespace MonoDevelop.UnitTesting.Expecto {
    public class ExpectoTestSuite : UnitTest {
        public ExpectoTestSuite(string name) : base(name) { }

        public ExpectoTestSuite(string name, WorkspaceObject ownerSolutionItem) : base(name, ownerSolutionItem) { }

        protected override UnitTestResult OnRun(TestContext testContext) {
            throw new NotImplementedException();
        }
    }
}
